client/keys.go

```diff
@@ -13,11 +13,9 @@ import (
 
 // PrivateKey represents a keyless-backed RSA private key.
 type PrivateKey struct {
-	public crypto.PublicKey
-	ski    gokeyless.SKI
-	digest gokeyless.Digest
-
-	client *Client
+	public   crypto.PublicKey
+	client   *Client
+	template *gokeyless.Operation
```

Comment:
do you like to create a new Metadata struct here? and call it md? Decouple from gokeyless.Operation? key.template just confuses me a little bit.